www/main/settings/components/Setting.jsx

```diff
@@ -131,6 +144,17 @@ const Setting = ({ settingObj, stateVar, stateFunction }) => {
         </>
       );
       break;
+    case 'reset-button':
+      settingDOM = (
+        <button
+          onClick={handleResetFontSizes}
+          className="icon-reset"
+          title={settingObj.tooltip}
+          style={{ backgroundImage: `url('assets/img/icons/${settingObj.icon}')` }}
+        >
```

Comment:
Add the text "Reset to Default"alongwith the icon in button. Instead of icon as background image, then you can add it as children element of button tag. 

Also, set cursor of button to pointer in css file.